product_docs/docs/edb_plus/39/04_using_edb_plus.mdx

```diff
@@ -72,13 +72,16 @@ For more information about the `pg_hba.conf` file, see the PostgreSQL core docum
 
  `scriptfile` is the name of a file residing in the current working directory, containing SQL and/or EDB\*Plus commands that will be automatically executed after startup of EDB\*Plus. `ext` is the filename extension. If the filename extension is `sql`, then the `.sql` extension may be omitted when specifying `scriptfile`. When creating a script file, always name the file with an extension, otherwise it will not be accessible by EDB\*Plus. (EDB\*Plus will always assume a `.sql` extension on filenames that are specified with no extension.)
 
+!!! note
+    When you run the commands in the following examples you may be using a newer version of EDB\*Plus and as such the EDB\*Plus build number shown in your output may be different. 
+
 The following example shows user `enterprisedb` with password `password`, connecting to database `edb` running on a database server on the `localhost` at port `5444`.
 
 ```text
 C:\Program Files\edb\as13\edbplus>edbplus enterprisedb/password
-Connected to EnterpriseDB 13.0.1 (localhost:5444/edb) AS enterprisedb
+Connected to EnterpriseDB 13.0.0 (localhost:5444/edb) AS enterprisedb
```

Comment:
Neither version 13.0.1 or 13.0.0 seem right for this message as I believe the version is supposed to correspond to EPAS 13.  The initial version of EPAS 13 was 13.1.4.  There seems to have been some error in updating the console output in some previous version.